packages/node-resolve/README.md

```diff
@@ -62,18 +62,12 @@ Default: `false`
 
 If `true`, instructs the plugin to use the `"browser"` property in `package.json` files to specify alternative files to load for bundling. This is useful when bundling for a browser environment. Alternatively, a value of `'browser'` can be added to the `mainFields` option. If `false`, any `"browser"` properties in package files will be ignored. This option takes precedence over `mainFields`.
 
-### `customResolveOptions`
+### `moduleDirectory`
```

Comment:
Despite the singular name, `moduleDirectory` can actually be an array in `resolve`. We use this extensively as we have some modules installed via bower, so we set this to `['node_modules', 'bower_components'].

Reply:
Thanks. Will make this clear

Reply:
pushed commit for this